src/plugins/unified_search/public/filter_bar/filter_item/filter_item.scss

```diff
@@ -18,15 +18,15 @@
   }
 }
 
-.globalFilterItem-isDisabled {
+.euiBadge.globalFilterItem-isDisabled {
   color: $euiColorDarkShade;
   background-color: transparentize($euiColorLightShade, .5);
   text-decoration: line-through;
   font-weight: $euiFontWeightRegular;
   font-style: italic;
```

Comment:
I couldn't see where this was actually showing the italic font (before or after this pr). Is it worth removing with this PR since it's touching these classes?

Reply:
Thanx @mdefazio, done https://github.com/elastic/kibana/pull/156639/commits/484f11dd726f8669ec0e6d54bb35c768cf1fe8c8